src/main/java/org/jitsi/jicofo/recording/Recorder.java

```diff
@@ -79,6 +79,12 @@ public void dispose()
     public abstract boolean isRecording();
 
     /**
+     * Process incoming packet.
+     * @param packet the packet to process.
+     */
+    public abstract void processIncomingPacket(Packet packet);
```

Comment:
Adding that in base class requires to review and update docs in al subclasses/check for potential issues. There must be assumption that packets may be processed not in the order they were received.